### DISCUSSION TITLE
Problem with motor in backward direction

### DISCUSSION THREAD

**Discussion by PabloMasa** [kachurovskiy/nanoels#206] [Q&A]
Hi everyone, im experiencing a little problem with my h4 and not sure whats is it about. 
everything works fine in one direction but when i try to go -z the motor get stuck o something. doesnt looks mechacanical problem because if  i change the direction with tthe driver jumper it do the same but i nthe other direction. 
im curently usin a nema34 with a CL86T driver, its closed loop. and a 2000 lpr encoder. any idea on what could be causing the problem? 
thanks in advance. 
Pablo.

**Comment by kachurovskiy** [comment]
Hi @PabloMasa, try turning off the closed loop mode and running the motor in the open loop mode. If it runs well, double-check the wiring from motor to the driver, it's a lot of wires that should be connected exactly right and it's easy to get them wrong.

![PXL_20231029_140822354](https://github.com/user-attachments/assets/7d8f15fa-284e-4d7a-b2e8-e067a61e156d)

Here's my wiring from a similar driver, perhaps it could give some hints though it doesn't necessarily has to match yours, check the motor papers.

**Comment by kachurovskiy** [comment]
Another possibility is that in your signal circuit is drawing too much current when signalling DIR HIGH lowering the signal level on the STEP slightly below the threshold. Try adding a 100 Ohm resistor inline on the DIR signal to test this hypothesis.

**Reply by PabloMasa** [reply]
so i have tried everything, doble check cable conections and tried a second encoder. i still cannot figured out whats the problem, the only new thing i have discovered is that the ASY mode works fine in both directions.
does it mean something?

**Reply by kachurovskiy** [reply]
@PabloMasa can you please post some pictures and videos of your setup? It's hard to make any conclusions without seeing what you've got.
